cumulus/polkadot-parachain/common/Cargo.toml

```diff
@@ -0,0 +1,25 @@
+[package]
```

Comment:
This entire crate is not really needed. If you want to have these aliases, declare them in the polkadot parachain binary.

Reply:
They are not required. I already assumed that you wanted to do this. For now we should focus on the omni node and node some non existing builder. So, either remove or move to the binary directly.

Reply:
Ok, done.

Reply:
> They are not required. I already assumed that you wanted to do this. For now we should focus on the omni node and node some non existing builder. So, either remove or move to the binary directly.

@bkchr I don't agree your opinion here, please hear me out:

In order to get out of the technical debt of the node side, we inevitably need `cumulus-service` or `omni-node-common` or similar to exist. 

A lot of the code between the 3 templates + `polkadot-parachain` is shared, but in reality is is copy-pasted because we don't have a shared crate. And reusable compoents (like, "_build an aura import queue for me please_") are the long hanging fruit way to make improvements to this.

I personally think it makes for a good learning objective for @serban300 to start thinking about such refactors. The time he would spend doing this refactor is well worth it + he can demonstrate that he can already simplify the code of the templates as well with the same traits and helpers that he has added to this PR.

I think your intention in asking to revert these refactors is speeding up the release of the omni-node itself, but I don't think this will speed it up by much. @serban300 and I already have [another branch ready with release-able changes](https://github.com/paritytech/polkadot-sdk/compare/master...kiz-omni-node), which will come right after this PR. The main reason I am holding back on the above branch is to do more testing with more teams etc, not coding or engineering effort. 

I won't insist on revering the changes in this PR per-se, but I hope we have a consensus to _encourage_ such refactors in the coming PRs related to omni-node, as it serves our long term goals.

Reply:
I would like to merge this PR in order to then continue the work related to renaming `polkadot-parachain` on top of it. But let's continue this discussion and use the conclusion in the following PRs